eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/SubmittedVariantsRestControllerTest.java

```diff
@@ -58,28 +63,43 @@
 
     private static final Long CLUSTERED_VARIANT = null;
 
-    private static final Boolean MATCHES_ASSEMBLY = null;
+    private static final Boolean SUPPORTED_BY_EVIDENCE = false;
+
+    private static final Boolean MATCHES_ASSEMBLY = true;
+
+    private static final Boolean ALLELES_MATCH = null;
+
+    private static final Boolean VALIDATED = null;
 
     @Test
     public void testGetVariantsRestApi() throws AccessionCouldNotBeGeneratedException {
-        List<AccessionResponseDTO> generatedAccessions = basicRestController.generateAccessions(
-                getListOfVariantMessages());
+        List<AccessionResponseDTO<SubmittedVariantDTO, ISubmittedVariant, String, Long>> generatedAccessions =
+                basicRestController.generateAccessions(getListOfVariantMessages());
         assertEquals(2, generatedAccessions.size());
         String accessions = generatedAccessions.stream().map(acc -> acc.getAccession().toString()).collect(
                 Collectors.joining(","));
         String getVariantsUrl = URL + accessions;
         ResponseEntity<List> getVariantsResponse = testRestTemplate.getForEntity(getVariantsUrl, List.class);
```

Comment:
If you declare this as `ResponseEntity<List<SubmittedVariantDTO>>` will ensure type safety and simplify the parsing, because you can access the objects instead of maps.